content/post/21-10-01-progress-report-27.md

```diff
@@ -0,0 +1,281 @@
++++
+title = "Progress Report #27: Transported to the Chess Dimension"
+author = "Space Wizards"
+date = 2021-09-29
+categories = [
+    "Progress Report"
+]
+reddit = ""
+thumbnail = "pr_27"
++++
+
+<!--more-->
+
+## Website Upgrade
+
+As you may have noticed, our official website has gotten a major facelift thanks to PJB. We also commissioned some nice new background art from AsPev as our old one was a random photo from NASA, and a little crusty. You can download a full resolution of it [here](https://cdn.discordapp.com/attachments/800430554662436944/873764090676379688/Commission_Swept_3.png).
+
+<!--
+TODO: Update this link with one from the website - update the website background - add comparison of the new website.
+-->
+
+## Updated Playtest Schedule
+
+We're moving playtests to a consistent schedule due to their popularity and developers wanting more feedback from the community on the state of the game. If you currently have access to Space Station 14 on Steam and aren't in our Discord server we HIGHLY RECOMMEND you join as it's the best place to stay informed about updates on the project.
+
+Two times a month on Fridays, with separate tests for US and EU timezones on those days.
+
+For EU it will be at [2PM UTC](https://dateful.com/convert/utc?t=2pm&tz2=CEST-Central-European-Summer-Time)
+For US it will be at [10PM UTC](https://dateful.com/convert/utc?t=10pm&tz2=CDT-CST-Central-Time)
+
+(That means since we had a playtest on October 1st the next playtest will be on October 15th).
+
+If you'd like to be notified when we playtest join our [Discord](https://discord.gg/MwDDf6t) and click the message in #rules-info to be pinged when we run tests.
+
+{{< imgw "/images/post/pr_27/tester" >}}
+
+## New Developer Docs
+
+Previously, we used to host all development-related documentation (technical documents, tutorials, vague and questional design dumps) on HackMD.
+
+HackMD had some nice attributes like live editing and being simple to use, however  it lacked in other areas like access management and navigation. Because of this, we have decided to start moving away from HackMD.
```

Comment:
```suggestion
HackMD had some nice attributes like live editing and being simple to use, however it lacked in other areas like access management and navigation. Because of this, we have decided to start moving away from HackMD.
```